bot/exts/moderation/infraction/infractions.py

```diff
@@ -76,9 +76,16 @@ async def kick(self, ctx: Context, user: FetchedMember, *, reason: t.Optional[st
         await self.apply_kick(ctx, user, reason)
 
     @command()
-    async def ban(self, ctx: Context, user: FetchedMember, *, reason: t.Optional[str] = None) -> None:
+    async def ban(
+        self,
+        ctx: Context,
+        user: FetchedMember,
+        duration: t.Optional[Expiry] = None,
+        *,
+        reason: t.Optional[str] = None
+    ) -> None:
         """Permanently ban a user for the given reason and stop watching them with Big Brother."""
```

Comment:
Docstring should be updated to correlate with this new feature. Using the ban command will not always end up as a permanent ban.

Reply:
Fixed in commit ea900a4.